corehq/motech/repeaters/forms.py

```diff
@@ -102,11 +102,27 @@ class FormRepeaterForm(GenericRepeaterForm):
         label="Include 'app_id' URL query parameter.",
         initial=True
     )
+    black_listed_users = forms.MultipleChoiceField(
```

Comment:
nit: rename 'blacklist' to 'blocklist'
```suggestion
    user_blocklist = forms.MultipleChoiceField(
```